app/views/welcome_mailer/welcome_email.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Why do we have these HTML, HEAD tags in the mailer view?
Should not they be in mailer layout and this view getting `yield` over there